### ISSUE TITLE
Inconsistent behavior of pausing the app and connecting via telnet

### ISSUE THREAD

**Issue by msurkovsky** [spirali/kaira#20]
According to documentation and usage, I'd expected that a usage **-b** and **-s** arguments work also for MPI versionl. But only share memory version works with those flags. Maybe MPI version should warn about that.

**Comment by spirali** [comment]
I agree this should be mentioned in the documentation. Btw: It is a long term goal (for long time) to implement this feature.
